sdk/nodejs/Makefile

```diff
@@ -35,8 +35,6 @@ build:: build_package build_plugin
 install_package::
 	cp dist/pulumi-resource-pulumi-nodejs "$(PULUMI_BIN)"
 	cp dist/pulumi-analyzer-policy "$(PULUMI_BIN)"
-	mkdir -p "$(PULUMI_BIN)/v6.10.2"
-	rm -rf "$(PULUMI_NODE_MODULES)/$(NODE_MODULE_NAME)/tests"
```

Comment:
Any idea why we were removing these in the first place?

Reply:
No idea.